### ISSUE TITLE
Wallaby misses test errors

### ISSUE THREAD

**Issue by DenisGorbachev** [wallabyjs/public#199]
https://github.com/fixin-platform/backend-worker/commit/f411aac3408a7045a635166e44d14586c6c5e6e1

Wallaby:
```
No failing tests, 3 passing
```

Mocha:
```
starfall@nx:~/workspace/backend-worker$ mocha


  bin/worker
    normal operation
      ✓ should run `echo` service
      1) should run `echo` service

  WorkerFactory
    normal operation
      ✓ should run `echo` service
    error handling
      ✓ should return error if message format doesn't match


  3 passing (248ms)
  1 failing

  1) bin/worker normal operation should run `echo` service:

      AssertionError: expected 1 to equal 0
      + expected - actual

      -1
      +0
      
    at ChildProcess.<anonymous> (test/bin/workerSpec.coffee:56:26)
    at maybeClose (child_process.js:766:16)
    at Process.ChildProcess._handle.onexit (child_process.js:833:5)
  



starfall@nx:~/workspace/backend-worker$ 
```

Actually, mocha reports "3 passing, 1 failing" — this is incorrect, as there are only 3 tests, so probably there's a structural error in my code (callbacks called twice, or whatever). But at least mocha outputs the assertion result, while Wallaby swallows it.

**Comment by ArtemGovorov** [comment]
> callbacks called twice, or whatever

Yes, it was the case. `spawnWorkerProcess` callback was calling `testDone` without error, so the wallaby thought the test was passing. A few moments later, `testDone` for the same test was called by `workerProcess.on "close"` callback, this time with an assertion error. Mocha is able to report it, wallaby ignored it because it think that the test is already finished.

> at least mocha outputs the assertion result, while Wallaby swallows it

Agreed, it's not good at all. The bug is fixed in core v1.0.80.

**Comment by DenisGorbachev** [comment]
Hmm, my Wallaby plugin is still at 1.0.24, and the "Update plugin" button in right-click menu is grayed out. Is there an issue with auto-update?

**Comment by fvanwijk** [comment]
Are notifications disabled? See https://github.com/wallabyjs/public/issues/178

**Comment by DenisGorbachev** [comment]
I've got them all enabled. Is it possible that I just missed that balloon?

![not](https://cloud.githubusercontent.com/assets/829578/8704807/24b59438-2b2e-11e5-83f0-88e6dbf47e24.png)

**Comment by ArtemGovorov** [comment]
Wallaby.js consists of two separately versioned parts - the [plugin and core](http://wallabyjs.com/docs/intro/how-it-works.html#plugin-and-core). 

Core is updated automatically, you can see its current version in the Wallaby Console when wallaby starts. 

Plugin asks if you'd like to install a new version via the balloon notification. If can't miss it, it'll hang there until you close it and even if you do, you'll still see it next time you restart your IDE.

The latest plugin version is currently 1.0.24 and latest core version is currently 1.0.80. The bug is fixed in the core v1.0.80, no plugin update was required to fix it.

**Comment by DenisGorbachev** [comment]
Ok, I see, thanks for explanation.
